java/com/google/security/wycheproof/TestUtil.java

```diff
@@ -96,4 +100,62 @@ private static void installOpenJDKProvider(String className) throws Exception {
     Provider provider = (Provider) Class.forName(className).getConstructor().newInstance();
     Security.insertProviderAt(provider, 1);
   }
+  
+  public static void printJavaInformation() {
+    System.out.println("Running with: ");
+    System.out.println("  java.runtime.name: " + System.getProperty("java.runtime.name"));
+    System.out.println("  java.runtime.version: " + System.getProperty("java.runtime.version"));
+  }
+  
+  /**
+   * Removes JDK crypto restriction.
+   * 
+   * Inspired by:
+   * https://github.com/jruby/jruby/blob/0c345e1b186bd457ebd96143c0816abe93b18fdf/core/src/main/java/org/jruby/util/SecurityHelper.java
+   */
+  public static void removeCryptoStrengthRestriction() {
+    try {
+      if (Cipher.getMaxAllowedKeyLength("AES") < 256) {
+        Class securityClass = Class.forName("javax.crypto.JceSecurity");
+        Field isRestricted = securityClass.getDeclaredField("isRestricted");
+        removeFinalModifier(isRestricted);
+        setFieldBooleanValue(isRestricted, false);
+        System.out.println("Successfully removed crypto strength restriction.");
+      }
+    } catch (ClassNotFoundException | IllegalAccessException | IllegalArgumentException | 
```

Comment:
Nit: we're following the Google Java Style Guide (https://google.github.io/styleguide/javaguide.html), this block can be rewritten as follows

`
try {
} catch (ClassNotFoundException
    | IllegalAccesException
    | IllegalArgumentException
    | NoSuchAlgorithmException
    | NoSuchFieldException
    | SecurityException ex) {
  System.out.println("Impossible to use unrestricted security policy with this JDK, reason: "
     + ex.getLocalizedMessage()); 
}
`